tests/generic_config_updater/test_ecn_config_update.py

```diff
@@ -94,8 +95,11 @@ def test_ecn_config_updates(duthost, ensure_dut_readiness, configdb_field, opera
                            "value": "{}".format(value)})
 
     try:
-        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
-        expect_op_success(duthost, output)
-        ensure_application_of_updated_config(duthost, configdb_field, ",".join(values))
+        if is_valid_platform_and_version(duthost, "WRED_PROFILE", "ECN tuning"):
```

Comment:
should we do this check in the beginning and skip the testcase if it is not supported platform/version?

Reply:
I initially planned to skip the testcase, but I received feedback that we should still test functionality to ensure that the platform validator works as expected and blocks GCU update to ConfigDB